README.rst

```diff
@@ -252,55 +264,55 @@ To configure any Fedora 20 bare metal machine (e.g. on OnMetal as above)::
+ * **redhat-openstack-novahost**: The server which has ``nova`` and other openstack administrative tools installed.
+ * **redhat-openstack-buildslave**: The server which will be created to run the ``redhat-openstack/centos-7`` builder.
 
-You'll need to add your public SSH key to the ``pistoncloud-jumphost``.
+You'll need to add your public SSH key to the ``redhat-openstack-jumphost``.
 A username and key for initial access to the jump host can be found in LastPass.
 Using that username and key, log into the jumphost and add your own public SSH key to the ``authorized_keys`` file of the jumphost user.
 
-Next log into the ``pistoncloud-novahost`` (credentials in LastPass) and add your own public SSH key.
+Next log into the ``redhat-openstack-novahost`` (credentials in LastPass) and add your own public SSH key.
 
 Finally, register your public SSH key with openstack by using the ``nova`` command, as follows:
 
 .. code-block:: console
 
-  [pistoncloud-novahost] $ cat > id_rsa_joe.blogs@clusterhq.com.pub
+  [redhat-openstack-novahost] $ cat > id_rsa_joe.blogs@clusterhq.com.pub
   ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC2imO7tTLepxqTvxacpNHKmqsRUdhM1EPdAVrBFadrYAC664LDbOvTqXR0iiVomKsfAe6nK9xZ5YzGFIpcOn/MeH45LOHVy5/+yx06qAnRkCDGZzQN/3qrs2K0v0L4XSIFbWmkFycAzG2phxFyAaJicK9XsJ9JaJ1q9/0FBj1TJ0CA7kCFaz/t0eozzOgr7WsqtidMrgrfrWvZW0GZR2PUc+1Ezt0/OBR8Xir0VGMgeLOrHprAF/BSK+7GLuQ9usa+nu3i46UuKtaVDMrKFCkzSdfNX2xJJYlRUEvLTa1VgswgL1wXXUwxXlDmYdwjF583CSFrVeVzBmRRJqNU/IMb joe.bloggs@clusterhq.com
 
-  [pistoncloud-novahost] $ nova keypair-add --pub-key id_rsa_joe.bloggs@clusterhq.com.pub clusterhq_joebloggs
+  [redhat-openstack-novahost] $ nova keypair-add --pub-key id_rsa_joe.bloggs@clusterhq.com.pub clusterhq_joebloggs
 
-Having done this, create or modify a ``~/.ssh/config`` file containing the aliases, usernames, hostnames for each of the servers and proxy commands that will allow direct access to the internal servers via the ``pistoncloud-jumphost``.
+Having done this, create or modify a ``~/.ssh/config`` file containing the aliases, usernames, hostnames for each of the servers and proxy commands that will allow direct access to the internal servers via the ``redhat-openstack-jumphost``.
 
 Here is an example of such a file::
 
-   Host pistoncloud-jumphost
+   Host redhat-openstack-jumphost
         User <jumphost_username>
         HostName <jumphost_public_hostname_or_ip_address>
 
-   Host pistoncloud-novahost
+   Host redhat-openstack-novahost
         User <novahost_username>
         HostName <novahost_public_hostname_or_ip_address>
-        ProxyCommand ssh pistoncloud-jumphost nc %h %p
+        ProxyCommand ssh redhat-openstack-jumphost nc %h %p
 
 With that ``~/.ssh/config`` content in place, run:
 
 .. code-block:: console
 
-   [laptop] $ fab -f slave/pistoncloud/fabfile.py create_server:clusterhq_joebloggs
+   [laptop] $ fab -H redhat-openstack-novahost -f slave/redhat-openstack/fabfile.py create_server:clusterhq_joebloggs
```

Comment:
I do have them hard-coded in `~/.ssh/config`, but with different names than are suggested here.

A better long-term option might be to get fabric to use a custom ssh-config that can be stored in last-pass, perhaps.